app/src/main/res/values/strings.xml

```diff
@@ -54,8 +44,11 @@
         - To set an image as representative of the category : click longer on the image and select the star icon -</string>
     <string name="Yes">Yes</string>
     <string name="No">No</string>
-    <string name="Close">Close</string>
-    <string name="numberOfSelectedPicturesText">%d selected pictures</string>
+    <plurals name="numberOfSelectedPicturesText">
+        <item quantity="zero">%d picture selected </item>
```

Comment:
```suggestion
```
Unlike in French, in English grammar "zero pictures" is considered plural. [Source](https://en.wikipedia.org/wiki/Grammatical_number#:~:text=english%20treats%20zero%20with%20the%20plural%20number.)